erdpy/tests/shared.sh

```diff
@@ -1,6 +1,10 @@
 export PYTHONPATH=$(realpath ../../)
 echo "PYTHONPATH = ${PYTHONPATH}"
 
+# if you want to test with the local erdpy you can uncomment this command
```

Comment:
Well, that is what `ERDPY="python3 -m erdpy.cli"` is for, in conjunction with `export PYTHONPATH=$(realpath ../../)`. Didn't that work as expected perhaps?

Reply:
removed this comment